addons/repair/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import sql
+from openupgradelib import openupgrade
+
+
+def update_repair_fee_invoice_relation(env):
+    openupgrade.logged_query(
+        env.cr, sql.SQL(
+            """UPDATE repair_fee rf
+            SET invoice_line_id = aml.id
+            FROM account_invoice_line ail
+            JOIN account_move_line aml ON aml.old_invoice_line_id = ail.id
+            WHERE rf.{} = ail.id"""
+        ).format(
+            sql.Identifier(openupgrade.get_legacy_name("invoice_line_id"))
+        ),
+    )
+
+
+def update_repair_line_invoice_relation(env):
+    openupgrade.logged_query(
+        env.cr, sql.SQL(
+            """UPDATE repair_line rl
+            SET invoice_line_id = aml.id
+            FROM account_invoice_line ail
+            JOIN account_move_line aml ON aml.old_invoice_line_id = ail.id
+            WHERE rl.{} = ail.id"""
+        ).format(
+            sql.Identifier(openupgrade.get_legacy_name("invoice_line_id"))
+        ),
+    )
+
+
+def update_repair_order_invoice_relation(env):
+    openupgrade.logged_query(
+        env.cr, sql.SQL(
+            """UPDATE repair_order ro
+            SET invoice_id = am.id
+            FROM account_invoice ai
+            JOIN account_move am ON am.old_invoice_id = ai.id
+            WHERE ro.{} = ai.id"""
+        ).format(
+            sql.Identifier(openupgrade.get_legacy_name("invoice_id"))
+        ),
+    )
+
+
+def update_repair_order_user_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE repair_order
```

Comment:
This is doing things on post, where the default has already acted, so this is probably not correct, depending on which is the default value (I assume that the current user, so all will be assigned to `admin`).

Reply:
IMO it's correct because `repair.order` exists previously in 12.0, right?

Reply:
I'm not talking about the table, but the field `user_id`.

Reply:
Ok, right, default `user_id` value auto assigned admin probably. Maybe a good idea is force `user_id` change to all records and remove `WHERE user_id IS NULL`.

Reply:
Exactly. That's what I'm talking about since the beginning.